src/EnergyPlus/DataSurfaces.hh

```diff
@@ -1821,9 +1826,14 @@ struct SurfacesData : BaseGlobalStruct
         this->Z0.deallocate();
         this->RepresentativeSurfaceMap.clear();
         this->AllHTSurfaceList.clear();
+        this->AllExtSolarSurfaceList.clear();
```

Comment:
370 lines of code could be removed just by changing the contents of this function to `*this = SurfacesData();` but I do get that not everyone is comfortable doing that, so it can be done later.